### PR TITLE
Fix deprecation format

### PR DESCRIPTION
**Pull request by edenhill**

### CODE REVIEW

File: kafka/config.go [hunk 1]
```diff
@@ -39,15 +39,15 @@ type ConfigValue interface{}
...
 // topic configuration properties shall be specified in the standard ConfigMap.
 // For backwards compatibility, "default.topic.config" (if supplied)
 // takes precedence.
 type ConfigMap map[string]ConfigValue
 
 // SetKey sets configuration property key to value.
 //
-// Deprecated: For user convenience a key prefixed with {topic}. will be
+// For user convenience a key prefixed with {topic}. will be
 // set on the "default.topic.config" sub-map, this use is deprecated.
```
Comment (rnpridgeon) [inline kafka/config.go]: Since `default.topic.config` has been deprecated this statement is no longer true right?

Comment (rnpridgeon) [reply kafka/config.go]: The  code below needs to be removed now that `default.topic.config` is no longer used.

```
	if strings.HasPrefix(key, "{topic}.") {
		_, found := m["default.topic.config"]
		if !found {
			m["default.topic.config"] = ConfigMap{}
		}
		m["default.topic.config"].(ConfigMap)[strings.TrimPrefix(key, "{topic}.")] = value
	} else {
		m[key] = value
	}
```

Comment (edenhill) [reply kafka/config.go]: Since both {topic} and default.topic.config are deprecated I think we can leave this as-is and remove them both in v2.0

Comment (rnpridgeon) [reply kafka/config.go]: Fair enough

**Pull request merged**